.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "3-lending/rai-test/RAI-Scrypto-Lending-Platform-PTE"]
```

Comment:
Why do you need to have a git submodule and why would it be placed in the root directory?

Reply:
I was developing the web frontend using repl.it platform since it automatically provides a hosted link for anyone to access - it was a nice feature that I found from doing the buildspace courses. 
As a result of working in repl.it, I had set up a separate git repo for the web frontend and didn't want to have 2 separate codebases for my submission vs the repl.it hosted link, so the best approach seemed to be to link to the source code of the web frontend using git submodules in my project submission repo.